tasks/example_processes/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I'm not sure if this format is really suitable. Maybe drop it for now? Or leave only "task_type"?